### ISSUE TITLE
-q, --quiet no longer suppresses output

### ISSUE THREAD

**Issue by kayleanelson** [TACC/Lmod#421]
We have upgrade to Lmod 8.1.14 and the `-q, --quiet` flag no longer suppresses output from module purge:

Version 7.8.15
```
$ module purge -q
$
```

Version  8.1.14
```
$ module purge -q
The following modules were not unloaded:
  (Use "module --force purge" to unload all):

  1) StdEnv
$
```

**Comment by rtmclay** [comment]
All options must be placed in standard unix fashion namely before the command:
```
$ module -q purge
```
There was a bug in purge when there were sticky modules and --quiet option.  This is fixed in Lmod 8.1.18.  Please test this new version when you get a chance.

**Comment by rtmclay** [comment]
O.K. to close this issue?
